docs/en/02_Developer_Guides/00_Model/10_Versioning.md

```diff
@@ -318,6 +329,47 @@ The ownership relationship is tracked through an `[image]` [shortcode](/develope
 which is automatically transformed into an `<img>` tag at render time. In addition to storing the image path,
 the shortcode references the database identifier of the `Image` object.
 
+### Changesets, a.k.a "Campaigns"
+
+Changes to many DataObjects can grouped together using the `ChangeSet` object, better known by its frontend name, "Campaign" (provided the `campaign-admin` module is installed). By grouping a series of content changes together as one cohesive unit, content editors can bulk publish or revert an entire body of content all at once, which affords them much more power and control over interdependent content types.
+
+Records can be added to a changeset in the CMS by using the "Add to campaign" button
+that is available on the edit forms of all pages and files. Programmatically, this is done by creating a `SilverStripe\Versioned\ChangeSet` object and invoking its `addObject(DataObject $record)` method.
+
+<div class="info" markdown="1">
+Any DataObject can exist in any number of changesets, and even added to a changeset in advance of being published. While a record need not have modifications to be part of a changeset, for practical purposes, changesets are only concerned with records that have modifications.
+</div>
+
+By including an object in a changeset, the user is implicitly including all of that object's owned records (as declared by the `$owns` setting). This ensures that when a changeset is published (or reverted), the action cascades through not only all of the items explicitly added to the changeset, but also all of the records that each of those items owns.
```

Comment:
It's worth adding a distinction and full explanation of the difference between "implicit" and "explicit" added items, and that all objects are either of the two flags. Items that are both explicitly and implicitly added are treated as explicit, which isn't obvious to the user.